src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable-next-line import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
 body {
+  box-sizing: border-box;
```

Comment:
better 

```
html {
  box-sizing: initial;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}
```

Reply:
The requirements for the task stated: don't use * selector (it impacts performance).